tools/rna_tools/vienna_rna/rnafold.xml

```diff
@@ -31,15 +31,19 @@
     #if $layout_type ==0
         --layout-type=$general_options.layout_type
     #end if
-    #if $mea:
-        --MEA=$mea
+    
+	#if $mea:		 
+        $mea=$meagamma	 
```

Comment:
Why this change? The old one called $mea was for the gamma value, isn't it?

Reply:
$mea is selective, meaning user wants to use mea mode instead of default mode. mea mode has a gamma parameter that the user can modify. The old one always invokes mea mode, which is not the default binary behavior.

Reply:
We could make the $mea (gamma) float just optional, so if this value is set, then the mode is choosen.